app/Jobs/FetchEnsDetails.php

```diff
@@ -35,6 +36,10 @@ public function __construct(
      */
     public function handle(): void
     {
+        Log::info('FetchEnsDetails Job: Processing', [
+            'wallet' => $this->wallet->address,
```

Comment:
also log the ENS name here and at the end so we can see differences between the two in case they changed it